### PR TITLE
fix acceptance test

### PR DESCRIPTION
**Pull request by cgardens**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -152,9 +146,6 @@ jobs:
           GIT_REVISION: ${{ github.sha }}
           CORE_ONLY: true
 
-      - name: Ensure no file change
```
Comment (cgardens) [inline .github/workflows/gradle.yml:155]: the point of the test_docker command is to run acceptance tests in docker. it shouldn't be duplicating build checks.

File: .github/workflows/gradle.yml [hunk 2]
```diff
@@ -216,9 +207,6 @@ jobs:
 #          GIT_REVISION: ${{ github.sha }}
 #          CORE_ONLY: true
 #
-#      - name: Ensure no file change
```
Comment (cgardens) [inline .github/workflows/gradle.yml:219]: same as above but for k8s.

File: airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java [hunk 3]
```diff
@@ -689,8 +687,11 @@ private JsonNode getDbConfig(PostgreSQLContainer psql, boolean hiddenPassword, b
     try {
       final Map<Object, Object> dbConfig = new HashMap<>();
 
+      // todo (cgardens) - hack to get building passing in CI. need to follow up on why this was necessary
+      // (and affect on the k8s version of these tests).
+      dbConfig.put("host", "localhost");
```
Comment (cgardens) [inline airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java:692]: empirically this is fixing it. I'm not entirely sure why yet, but biasing towards getting it fixed for now. likely this is a change in the interaction between docker networking (need to try setting my local docker version to the one being used by github actions), test containers, and/or some other component of the github actions environment.

File: .github/workflows/gradle.yml [hunk 4]
```diff
@@ -68,12 +68,6 @@ jobs:
       - name: Generate Template scaffold
         run: ./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates --scan
 
-      - name: Format
```
Comment (cgardens) [inline .github/workflows/gradle.yml:71]: this is increasing the build time by between 25-50%. doesn't seem worth it. 

while it allows us to identify a somewhat common problem earlier, it is ultimately one that's really easy to fix, while it delays feedback on ones that are hard to fix. this is bad for our attention. i can fix a formatting problem that someone else pushes in my sleep. for other build failures, really the person who pushed the change is best suited to fix it but they way to so long that they don't notice the problem until way later.

**Review by jrhizor — APPROVED** [review]

File: .github/workflows/gradle.yml (hunk 4 above)
Comment (cgardens) [reply .github/workflows/gradle.yml]: this caused more build problems. going to try to fix separately.

**Pull request merged**